bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -279,6 +289,8 @@
    //
    private TourData                                         _tourData;
    private TourChartConfiguration                           _tcc;
+
+   private Long                                             _currentTourId_for_CustomTracks = null;
```

Comment:
There is no need to initialize a field with `null` or `false`, it is already `null` or `false` but in my old code there can still be the `null` field asignments